ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committing/ProcessTransactionSubmission.scala

```diff
@@ -486,14 +511,20 @@ private[kvutils] class ProcessTransactionSubmission(
       .get(stateKey)
       .flatMap {
```

Comment:
Can you remove this nested `.flatMap` logic, please? I.e., you could rewrite it with for comprehension for better readability.

Reply:
Fixed in 14a595f.